CheatSheets/nodeJs-cheatsheet.md

```diff
@@ -0,0 +1,19 @@
+## NodeJS
+
+| Command | Description |
+| ------- | ----------- |
+| `node` | Run the Node REPL in your terminal |
+| `node —version` | Print your current Node version |
+| `node filename.js` | Execute the Node code in filename.js |
+
+## Built In Methods
+
+| `fs` | read and write files on your file system |
+| `path` | combine paths regardless of which OS you're using |
+| `http` | make requests and create HTTP servers |
+| `https` | work with secure HTTP servers using SSL/TLS |
+| `events` | work with the EventEmitter |
+| `crypto` | cryptography tools like encryption and hashing |
+| `process` | information about the currently running process, e.g. process.argv for arguments passed in or process.env for environment variables | 
```

Comment:
The table is broken.

Reply:
@crescentpartha Can you please explain more about NodeJS Cheatsheet. So I can change based on that.

Thanks!

Reply:
- [node-cheatsheet](https://courses.cs.washington.edu/courses/cse154/19su/resources/assets/cheatsheets/node-cheatsheet.pdf) 
- [https://overapi.com/nodejs](https://overapi.com/nodejs) 
- [Node.js Documentation2](https://nodejs.org/api/documentation.html#json-output) 
- [Node.js Documentation1](https://nodejs.org/api/path.html#path_path_normalize_p)
You can check out those resources. then you create a structure in your own way.